packages/iframe-coordinator-cli/package.json

```diff
@@ -23,6 +23,11 @@
     "typescript": "^4.8.3",
     "vite": "^3.1.3"
   },
+  "files": [
+    "dist",
```

Comment:
the `dist` folder was not getting published by default as it is in the .gitignore file.